.travis.yml

```diff
@@ -17,15 +17,14 @@ jobs:
 before_install:
   - sudo apt-get -y install mandoc
   - pip install -U pip
-  - python setup.py install
```

Comment:
This should not be removed, but moved to the `install:` section. 

Currently this works because the setup doesn't do anything special that the tests would need and dependencies are installed from `requirements.txt`. However, by adding this line you execute the setup. If it fails the tests will fail, which is a good thing.

Reply:
You're somewhat right:

The original problem was that the tests were using this version of htcrystalball that was installed by this line. All the code executed was in that install, which is why the coverage of the `htcrystalball` folder was 0%. We weren't using our repo's copy.

Adding `__init__.py` to the tests dir fixes that, as it forces the import to look in the current repo dir first.

However, I do think it's valuable to test the installation procedure. So I will reintroduce this line, but at the end of the tests. It's a test now, rather than a dependency.